webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <h5 class="title spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </h5>
+      <ul :class="connectionsClass">
+        <li
+          v-for="connection in filteredConnections"
+          :key="connection.id"
+          class="connections__item"
+        >
+          <user-teaser :user="connection" />
+        </li>
+      </ul>
+      <base-button
+        v-if="hasMore"
+        :loading="loading"
+        class="spacer-x-small"
+        size="small"
+        @click="$emit('fetchAllConnections', type)"
+      >
+        {{
+          $t('profile.network.andMore', {
+            number: allConnectionsCount - connections.length,
+          })
+        }}
+      </base-button>
+      <ds-input
+        v-if="!hasMore"
+        v-focus="true"
+        :name="`${type}Filter`"
+        :placeholder="filter"
+        class="spacer-x-small"
+        icon="filter"
+        size="small"
+        @input.native="setFilter"
+      />
+    </template>
+    <p v-else class="nobody-message">{{ userName }} {{ $t(`profile.network.${type}Nobody`) }}</p>
+  </base-card>
+</template>
+
+<script>
+import { escape } from 'xregexp/xregexp-all.js'
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-26T18:07:10Z" title="Thursday, March 26th 2020, 7:07:10 pm +01:00">Mar 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,157 @@
+<template>
+  <base-card class="hc-connections">
+    <ds-space v-if="this.connections && this.connections.length" margin="x-small">
+      <ds-text tag="h5" color="soft">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </ds-text>
+    </ds-space>
+    <template v-else>
+      <p class="no-connections">{{ userName }} {{ $t(`profile.network.${type}Nobody`) }}</p>
+    </template>
+    <template v-if="this.connections && this.connections.length <= 7">
+      <ds-space v-for="follow in uniq(this.connections)" :key="follow.id" margin="x-small">
+        <!-- TODO: find better solution for rendering errors -->
+        <client-only>
+          <user-teaser :user="follow" />
+        </client-only>
+      </ds-space>
+      <ds-space v-if="this.allConnectionsCount - this.connections.length" margin="small">
+        <base-button
+          @click="fetchConnections"
+          :loading="this.isLoading"
+          size="small"
+          color="softer"
+        >
+          {{
+            $t('profile.network.andMore', {
+              number: this.allConnectionsCount - this.connections.length,
+            })
+          }}
+        </base-button>
+      </ds-space>
+    </template>
+    <template v-else-if="this.connections.length > 7">
+      <div class="overflow-container">
+        <ds-space
+          v-for="follow in uniq(this.filteredConnections)"
+          :key="follow.id"
+          margin="x-small"
+        >
+          <client-only>
+            <user-teaser :user="follow" />
+          </client-only>
+        </ds-space>
+      </div>
+      <ds-space margin="x-small">
```


we are trying to refactor out all uses of `ds-space`, would you mind helping us with that?

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-03-30T08:09:38Z" title="Monday, March 30th 2020, 10:09:38 am +02:00">Mar 30, 2020</time>_

---


changed in a6484e3